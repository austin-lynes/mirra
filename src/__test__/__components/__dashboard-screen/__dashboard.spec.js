import React from 'react';
import { shallow } from 'enzyme'
import DashboardScreen from '../../../components/dashboard-screen';

let dashboardScreen;

describe('components/welcome-screen', () => {

    beforeAll(() => {
        dashboardScreen = shallow(<DashboardScreen />)
    })
    it('renders properly', () => {
        const instance = dashboardScreen.instance();
        expect(instance).toMatchSnapshot();
    })
})