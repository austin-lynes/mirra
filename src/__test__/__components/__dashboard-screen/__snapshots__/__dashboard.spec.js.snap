// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/welcome-screen renders properly 1`] = `null`;
